contracts/test/integration/IntegrationTestPriceBoundPSM.t.sol

```diff
@@ -92,6 +99,22 @@ contract IntegrationTestPriceBoundPSMTest is DSTest {
             rariVoltPCVDeposit
```

Comment:
feels like we should spin up another PCV Deposit and not use this deprecated one